modules/ui/src/main/scala/lucuma/ui/optics/package.scala

```diff
@@ -52,5 +57,18 @@ def unsafeDisjointOptionZip[S, A, B](
     (s: S) => l2.replace(ab.map(_._2))(l1.replace(ab.map(_._1))(s))
   )
 
-def optionIso[A, B](iso: Iso[A, B]): Iso[Option[A], Option[B]] =
-  Iso[Option[A], Option[B]](_.map(iso.get))(_.map(iso.reverseGet))
+extension [A, B](iso: Iso[A, B])
+  def option: Iso[Option[A], Option[B]] =
+    Iso[Option[A], Option[B]](_.map(iso.get))(_.map(iso.reverseGet))
+
+val OptionNonEmptyStringIso: Iso[Option[NonEmptyString], String] =
```

Comment:
Do you get an empty string from a `None`?

Reply:
Yup.